lib/process-services/src/lib/process-list/components/process-list/process-list.component.ts

```diff
@@ -275,6 +280,10 @@ export class ProcessInstanceListComponent extends DataTableSchema implements OnC
         this.rowClick.emit(this.currentInstanceId);
     }
 
+    onRowCheckboxToggle(event: CustomEvent) {
```

Comment:
mb we should add unit tests for new functionality?

Reply:
Added, thank you!